src/afl-fuzz-state.c

```diff
@@ -565,6 +567,26 @@ void read_afl_environment(afl_state_t *afl, char **envp) {
 
             }
 
+          } else if (!strncmp(env, "AFL_FUZZER_STATS_UPDATE_INTERVAL",
+
+                              afl_environment_variable_len)) {
+
+            u64 stats_update_freq_sec =
+                strtoull(get_afl_env(afl_environment_variables[i]), NULL, 0);
+            if (ULLONG_MAX == stats_update_freq_sec ||
```

Comment:
if this is ULLONG_MAX - 1 and is then multiplied by 1000 then we have an overflow right? better `if >= UINT_MAX`?

Reply:
Yeah, correct. Fixed.